### DISCUSSION TITLE
大佬，播放窗口能不能优化一下？

### DISCUSSION THREAD

**Discussion by ZHUOLUERNV** [cotaku/Bibi-Release#31] [Ideas]
![01](https://github.com/cotaku/Bibi-Release/assets/141299887/61a13f42-321b-4950-add8-e138a941141b)

![02](https://github.com/cotaku/Bibi-Release/assets/141299887/1ff7a09f-3eef-48f4-9f9a-4db1c88f15e1)

不喜欢全屏观看，但图1这播放窗口大黑边实在看着难受，如果改成图2这种就好了，跪求！

**Comment by cotaku** [comment]
可以，看来需要针对窗口最大化模式进行一些调整

**Reply by ZHUOLUERNV** [reply]
目前的版本全屏观看没有问题，只是窗口播放（包括横屏和竖屏视频）显示大黑边，而且窗口视频播放也不能鼠标左右调节音量亮度，而且也没有弹幕设置和开关按钮（全屏播放可以），就我本人来说比较喜欢窗口播放，因为能看到评论区和视频介绍，全屏播放也影响微信等消息的及时查看。

如果可以我还有个小小的请求，希望能显示视频评论区B友们的P地址。

官方B站PC客户端界面难看播放卡顿，表现十分拉胯，无奈现在真正好用的、能用的B站第三方PC客户端屈指可数，我找了很久，下载了3个B站第三方PC客户端，留下了2个，除了大佬你这个，还有个图2那个WILIWILI

https://github.com/xfangfang/wiliwili/releases/tag/v1.2.0

那个看界面应该是索尼掌机版本移植过来的，说实话界面、播放流畅度等等也不错，但可能是移植版本没针对PC端进行优化，缺失了很多基本的功能，比如连我关注了哪些UP主都看不了，所以我只能放弃。

你做的这个B站PC客户端可以说是目前最优质的版本了，播放流畅界面友好功能齐全，比官方好太多了，是PC端爱好者最后的灯塔，希望能一直维护下去，感谢你的回复，期待新版本上线！

**Reply by cotaku** [reply]
窗口播放是简化版的播放器，功能肯定没有完整的播放页面多，像鼠标快捷键和各种设置按钮之类的不想再写一遍代码了，预先调整好吧……
另外显示IP地址这种反人类功能肯定不会加的><

**Comment by cotaku** [comment]
增加了个选项：
![image](https://github.com/cotaku/Bibi-Release/assets/2822574/17455cf2-0d1a-49c1-9422-357b7423de40)
